src/util/fetch/fetch.js

```diff
@@ -34,6 +34,21 @@ export const post = (endpoint, data) =>
       });
   });
 
+  /** Fetch endpoint and delete post
+   * @param {Endpoint} endpoint 
+   */
+export const remove = endpoint => {
```

Comment:
you don't need the outer set of {}, you can do endpoint=>new Promese(... directly